usagov/vote-gov-drupal
# VOTE-2952: Delete deprecated pdf link field description

[VOTE-2952](https://cm-jira.usa.gov/browse/VOTE-2952)

Removes deprecated pdf link. This should be merged in after the data has been migrated on prod.

## Deployment and testing

### QA/Testing instructions

1. lando retune
2. Navigate to http://vote-gov.lndo.site/admin/structure/block-content/manage/registration_form_selector/fields
3. Confirm there is only one field for the pdf link. (field_description should not exist)

## Checklist for the Developer




## Checklist for the Peer Reviewers
